frontend/src/buttons/DeleteButton/DeleteButton.tsx

```diff
@@ -27,13 +28,15 @@ export const DeleteButton: React.FC<Props> = ({ device }) => {
     tooltip = 'Device must be offline'
   }
   if (device.shared) {
-    disabled = true
-    tooltip = 'You must be the device owner'
+    disabled = false
+    deleteDevice  = false
+    tooltip = 'Leave Device'
+    warning = "Are you sure to leave this device?"
```

Comment:
@JamieRuderman @vinson-remoteit could you help me with this message of confirmation?